casepro/cases/tests.py

```diff
@@ -291,6 +292,38 @@ def test_get_all(self):
         self.assertEqual(set(Case.get_closed(self.unicef)), {case2})
         self.assertEqual(set(Case.get_closed(self.unicef, user=self.user1, label=self.pregnancy)), {case2})
 
+    def test_get_all_passed_response_time(self):
+        bob = self.create_contact(self.unicef, 'C-002', "Bob")
+        cat = self.create_contact(self.unicef, 'C-003', "Cat")
+        nic = self.create_contact(self.nyaruka, 'C-104', "Nic")
+
+        msg1 = self.create_message(self.unicef, 123, self.ann, "Hello 1", [self.aids])
+        msg2 = self.create_message(self.unicef, 234, bob, "Hello 2", [self.aids, self.pregnancy])
+        msg3 = self.create_message(self.unicef, 345, cat, "Hello 3", [self.pregnancy])
+        msg4 = self.create_message(self.nyaruka, 456, nic, "Hello 4", [self.code])
```

Comment:
You are right, I removed the cases they were reference from but missed them. Thanks